cmd/certificates/sign.go

```diff
@@ -5,59 +5,57 @@ package main
 
 import (
 	"crypto/x509"
+	"fmt"
+	"log"
 
 	"github.com/spf13/cobra"
 	"github.com/zeebo/errs"
 
-	"storj.io/storj/pkg/cfgstruct"
 	"storj.io/storj/pkg/identity"
 )
 
 var (
 	signCmd = &cobra.Command{
-		Use:   "sign",
+		Use:   "sign [signee identity-dir]",
 		Short: "Sign a CA and update corresponding CA and identity certificate chains",
+		Args:  cobra.ExactArgs(1),
 		RunE:  cmdSign,
 	}
 
 	signCfg struct {
-		CA       identity.FullCAConfig
-		Identity identity.Config
+		SigneeCACfg    identity.PeerCAConfig
+		SigneeIdentCfg identity.PeerConfig
 		// NB: defaults to same as CA
 		Signer identity.FullCAConfig
 	}
 )
 
-func init() {
-	rootCmd.AddCommand(signCmd)
-	cfgstruct.Bind(signCmd.Flags(), &signCfg, cfgstruct.ConfDir(defaultConfDir))
-}
-
 func cmdSign(cmd *cobra.Command, args []string) error {
-	ca, err := signCfg.CA.Load()
+	ca, err := signCfg.SigneeCACfg.Load()
 	if err != nil {
 		return err
 	}
 
-	ident, err := signCfg.Identity.Load()
+	ident, err := signCfg.SigneeIdentCfg.Load()
 	if err != nil {
-		return err
+		log.Printf("unable to load identity: %s\n", err.Error())
 	}
 
 	signer, err := signCfg.Signer.Load()
 	if err != nil {
+		fmt.Println("two")
 		return err
 	}
 	restChain := []*x509.Certificate{signer.Cert}
 
 	// NB: backup ca and identity
-	err = signCfg.CA.SaveBackup(ca)
+	err = signCfg.SigneeCACfg.SaveBackup(ca)
 	if err != nil {
 		return err
 	}
-	err = signCfg.Identity.SaveBackup(ident)
+	err = signCfg.SigneeIdentCfg.SaveBackup(ident)
 	if err != nil {
-		return err
+		log.Printf("unable to save backup of identity: %s\n", err.Error())
```

Comment:
why is the error on line 54 returned but the error here is just logged out? Don't we want the same behavior for each `SaveBackup` error?

Reply:
the thought was to allow for the signing of a ca without an identity having to be present. You could always generated an identity after signing.the CA.

Reply:
If we don't have a SigneeIdentCfg specified can we avoid logging at all? The message might look confusing.

Reply:
:+1: